<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>291</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_4">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>93</width>
          <height>418</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_11">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_17">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_16">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_15">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_5">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>273</width>
          <height>165</height>
         </rect>
        </property>
        <attribute name="label">
         <string>家人</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>273</width>
          <height>165</height>
         </rect>
        </property>
        <attribute name="label">
         <string>朋友</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>273</width>
          <height>165</height>
         </rect>
        </property>
        <attribute name="label">
         <string>黑名单</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_6">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>百度</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>谷歌</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QPushButton" name="btn_scrollArea">
      <property name="text">
       <string>scrollArea</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_toolBox">
      <property name="text">
       <string>Tool Box</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_tabWidget">
      <property name="text">
       <string>Tab Widget</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Wangwuhong</string>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>421</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_color">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择字体颜色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_debug">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>141</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
